app/views/events/admin_show.html.erb

```diff
@@ -4,7 +4,7 @@
 <section class="box">
   <h2 class="box-title">Applications</h2>
   <div class="detail-pair">
-    <%= pluralize(@event.applications.count, "application") %> currently
+    <span class="detail-pair-count"><%= pluralize(@event.applications.count,"application") %></span> currently
```

Comment:
Still in progress. I wasn't able to add the color to the count only since "application" is also in the function and I can't seem to figure out how to exclude it. So for now the count + application(s) is colored.